### PR TITLE
Fixes Google Scholar for Saved Lists

### PR DESCRIPTION
**Pull request by adam3smith**
Removes last test because URL results in 404 -- I don't think we can save those.

@zuphilip please review when you get a chance.

### CODE REVIEW

File: Google Scholar.js [hunk 1]
```diff
@@ -537,7 +536,13 @@ function scrapeAll(doc, itemObjs) {
 		ZU.doGet(item.citeUrl, function(text) {
 			var m = text.match(/href="((https?:\/\/[a-z\.]*)?\/scholar.bib\?[^"]+)/);
 			if (!m) {
+				//Saved lists and possibly other places have different formats for BibTeX URLs
+				//Trying to catch them here (can't add test bc lists are tied to google accounts)
 				Zotero.debug(text);
+				m = text. match(/href="(.+?)">BibTeX<\/a>/);
```
Comment (zuphilip) [inline Google Scholar.js]: Delete the space after the `.` and before `match`.

**Comment by zuphilip** [comment]
Yes, this looks good for me, see the comment above. Maybe you want to clean-up a little, i.e. delete the debug line and add the empty line 428 again.

**Comment by adam3smith** [comment]
OK, thanks -- included your changes & comments, removed the debug comment -- I didn't re-add the deleted line; it didn't make any sense to me.

**Comment by zuphilip** [comment]
@adam3smith Did you exclude the last replacement on line 499 of https://github.com/zotero/translators/pull/1113/commits/9c1b78f2c640e992f06728241ea6d0fce6c5b60c on purpose?
```diff
 		if(attachmentFrag.indexOf('/citations?') != -1) {
 			attachmentFrag = null;
 			//build attachment link when importing from saved citations
-			var caseId = ZU.xpathText(item.result, '(.//div[@class="gs_fl"]\
+			var caseId = ZU.xpathText(item.result, '(.//div[contains(@class, "gs_fl")]\
 				/a[contains(@href,"cites=") or contains(@href,"about=")]/@href)[1]');
 			if(caseId) caseId = caseId.match(/\b(?:cites|about)=(\d+)/);
 			if(caseId) caseId = caseId[1];
```

**Comment by adam3smith** [comment]
no, fixed now.

**Pull request merged**